src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -594,4 +617,16 @@ public String toString() {
       return name().toLowerCase(Locale.ROOT);
     }
   }
+
+  public static class NoStackTraceConnectException extends RuntimeException {
```

Comment:
maybe we can call this class `ReportingException`?. the exception doesn't have anything do with `ConnectException`.